File: Lab2.5/Lab2.2/Circle.cs
using System;

namespace GeometryLib
{
    public class Circle : Figure
    {
        private double radius;

        public Circle()
        {
            radius = 1.0;
        }

        public Circle(double radius)
        {
            this.radius = radius;
        }

        public double GetRadius() => radius;

        public override double GetArea()
        {
            return Math.PI * radius * radius;
        }

        public override double GetPerimeter()
        {
            return 2 * Math.PI * radius;
        }
    }
}

File: Lab2.5/Lab2.5/Circle.cpp
#include "Circle.h"
#define _USE_MATH_DEFINES
#include <cmath>

Circle::Circle() : radius(1.0) {}

Circle::Circle(double radius) : radius(radius) {}

double Circle::getRadius() const { return radius; }

double Circle::getArea() const {
    return M_PI * radius * radius;
}

double Circle::getPerimeter() const {
    return 2 * M_PI * radius;
}
